themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss

```diff
@@ -0,0 +1,26 @@
+@charset 'UTF-8';
+
+.opportunity-filter-hero-container {
+  h1 {
+    margin: 0;
+  }
+
+  .su-hero__card {
+    @include grid-media-min('lg') {
+      max-width: 650px;
+      text-align: center;
+      opacity: 0.9;
+    }
+  }
+}
+
+.opportunity-filter-search-container {
+  background-color: #f4f4f4;
+  border-bottom: 1px solid #e5e5e5;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.25);
+  margin-bottom: 72px;
+
+  .opportunties-list {
+    @include centered-column;
+  }
+}
```

Comment:
Currently this container styled the entire react components. However, the [mock](https://www.figma.com/file/CpblbZldiGM6oYawa9aB4Q/Look-%26-Feel?node-id=380%3A2207) up shows that the background only appears behind the SelectLists, excluding the Slugs.

Something we can review during our meeting after lunch tomorrow? 😄

Reply:
After thinking and sleeping on this, I decided to move the styling into the React Components.